README.md

```diff
@@ -164,6 +164,17 @@ For some components, external sources can be used instead.
 | `-DTHEROCK_RCCL_SOURCE_DIR=<PATH>`              | Path to rccl sources                           |
 | `-DTHEROCK_RCCL_TESTS_SOURCE_DIR=<PATH>`        | Path to rccl-tests sources                     |
 
+Further flags allow to build components with specific features enabled.
+
+| Other flags                 | Description                                                              |
+| --------------------------- | ------------------------------------------------------------------------ |
+| `-DTHEROCK_ENABLE_MPI=OFF`  | Enables building components with Message Passing Interface (MPI) support |
```

Comment:
Oh I was thinking about just adding to https://github.com/ROCm/TheRock/blob/main/docs/development/dependencies.md for now. That's lower visibility than the root README.md which seems appropriate for now.

I guess it's fine to add to here too.

Reply:
That document rather has what TheRock ships as additional dependencies which it can bundle (or consume from the system) thus it didn't seemed appropriate for that particular option.

Reply:
Okay, let's follow up on this. Maybe want to add Fortran to this as well?